pydatalab/pydatalab/apps/chat/blocks.py

```diff
@@ -60,7 +99,7 @@ def render(self):
                 {
                     "role": "user",
                     "content": f"""Here is the JSON data for the current item(s): {info_json}.
-Start with a friendly introduction and give me a one sentence summary of what this is (not detailed, no information about specific masses). """,
+                    Start with a friendly introduction and give me a one sentence summary of what this is (not detailed, no information about specific masses). """,
```

Comment:
```suggestion
Start with a friendly introduction and give me a one sentence summary of what this is (not detailed, no information about specific masses). """,
```

Again whitespace inside multiline strings starts at the edge of the file (col:0) (a bit of a Python oddity) -- it will count as several tokens(?) so we should avoid it.